src/utils/__tests__/csvValidation.test.ts

```
@@ -246,11 +258,15 @@ describe('csvValidation', () => {
         status: 'Unknown',
         kit_category: 'NotValid',
         datePurchased: '0242-06-04', // Invalid year (should be fixed)
+        generalNotes: 'a'.repeat(2000), // Too long
```

Biome (1.9.4)

[error] This property value named generalNotes is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named generalNotes


(lint/suspicious/noDuplicateObjectKeys)